projects/common/src/user/user.service.ts

```diff
@@ -0,0 +1,33 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { UserTraits } from '../telemetry/telemetry';
+import { InMemoryStorage } from './../utilities/browser/storage/in-memory-storage';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserService {
```

Comment:
Please rename to UserInfoService or something else. The name `UserService` may easily be confused that it relates to the backend "Hypertrace User Service".

https://hus.concierge.stage.razorpay.in/swagger/index.html